Sejima/Source/MUTime/MUTime.swift

```diff
@@ -0,0 +1,269 @@
+//
+//  MUTime.swift
+//  Sejima
+//
+//  Created by Loïc GRIFFIE on 31/10/2018.
+//  Copyright © 2018 Loïc GRIFFIE. All rights reserved.
+//
+
+import UIKit
+
+/// Class that provide a circular indicator to represent time with customizable options.
+@IBDesignable
+open class MUTime: MUNibView {
+
+    @IBOutlet private var timeLabel: MULabelCounter!
+    @IBOutlet private var timeBackground: UIView!
+
+    /// The progress value text font.
+    @objc open dynamic var timeFont: UIFont = .systemFont(ofSize: 13, weight: .bold) {
+        didSet {
+            timeLabel.textFont = timeFont
+        }
+    }
+
+    /// The progress value text color.
+    @IBInspectable open dynamic var timeColor: UIColor = .clear {
+        didSet {
+            timeLabel.textColor = timeColor
+        }
+    }
+
+    /// The progress value text background color.
+    @IBInspectable open dynamic var timeBackgroundColor: UIColor = .clear {
```

Comment:
Can be an override of backgroundColor and the superview will be always .clear and we just change timeBackground UIView?